presto-main/src/main/java/com/facebook/presto/operator/scalar/ArrayGreaterThanOrEqualOperator.java

```diff
@@ -49,18 +48,19 @@ private ArrayGreaterThanOrEqualOperator()
     @Override
     public FunctionInfo specialize(Map<String, Type> types, int arity, TypeManager typeManager, FunctionRegistry functionRegistry)
     {
-        Type type = types.get("T");
-        type = typeManager.getParameterizedType(StandardTypes.ARRAY, ImmutableList.of(type.getTypeSignature()), ImmutableList.of());
+        Type elementType = types.get("T");
+        Type type = typeManager.getParameterizedType(StandardTypes.ARRAY, ImmutableList.of(elementType.getTypeSignature()), ImmutableList.of());
         TypeSignature typeSignature = type.getTypeSignature();
-        return operatorInfo(GREATER_THAN_OR_EQUAL, RETURN_TYPE, ImmutableList.of(typeSignature, typeSignature), METHOD_HANDLE.bindTo(type), false, ImmutableList.of(false, false));
+        MethodHandle lessThanFunction = functionRegistry.resolveOperator(LESS_THAN, ImmutableList.of(elementType, elementType)).getMethodHandle();
```

Comment:
@cberner I just found OperatorType.LESS_THAN_OR_EQUAL is not enough for lessThanOrEqual
array1: ARRAY[1, 2, 3, 4]
array2: ARRAY[1, 2, 4, 3]
for the first 3 elements, array1[element] <= array2[element], but checking the last element will make (array1 <= array2) return false
while, actually, array1 < array2, (array1 <= array2) is true

OperatorType.LESS_THAN is enough for lessThanOrEqual

I am planning to do greaterThan and lessThan, and make lessThanOrEquals and greaterThanOrEquals calling them

What do you think?